lib/private-libwebsockets.h

```diff
@@ -39,6 +39,8 @@
 #endif
 
 #if defined(WIN32) || defined(_WIN32)
+#define WINVER 0x0601
```

Comment:
You're setting minimum version to windows 7. This is very restrictive. Why is this needed at all?

https://msdn.microsoft.com/en-us/library/windows/desktop/aa383745(v=vs.85).aspx#setting_winver_or__win32_winnt